static/js/apps/explore/success_result.tsx

```diff
@@ -56,11 +56,28 @@ interface SuccessResultPropType {
   userMessage: UserMessageInfo;
 }
 
+// Whether or not to show topic list in the user message.
+function showTopicsInUserMsg(pageMetadata: SubjectPageMetadata): boolean {
+  for (const category of pageMetadata.pageConfig.categories) {
+    for (const block of category.blocks) {
+      for (const column of block.columns) {
+        for (const tile of column.tiles) {
+          if (tile.type !== "PLACE_OVERVIEW") {
+            return false;
+          }
+        }
+      }
+    }
+  }
+  return true;
```

Comment:
nit: should we do:
if (#categories == 1 && #block-in-category == 1 && #column-in-block == 1 && #tile-in-column == 1 && tile.type == "PLACE_OVERVIEW") return true;

?

Reply:
oh, that's nicer, yes, updated!